bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx98-mobile-de.html

```diff
@@ -23,6 +23,9 @@
 
 {% block site_header %}{% endblock %}
 
+{% set play_store_link = play_store_url('firefox') + '&utm_source=WNP&utm_campaign=WNP98DE' %}
+{% set app_store_link = app_store_url('firefox') %}
```

Comment:
question (potentially blocking): do we need utm params on the app store links as well?

Reply:
Only the Play Store as far as I understand from the [issue](https://github.com/mozilla/bedrock/issues/11211#issuecomment-1055079275). I'm not sure if apple supports these GA type params in the app store.

As for the click events, yes these are already set up.